lib/plugins/aws/deploy/compile/events/apiGateway/lib/cors.js

```diff
@@ -9,6 +9,8 @@ module.exports = {
     _.forEach(this.validated.corsPreflight, (config, path) => {
       const resourceName = this.getResourceName(path);
       const resourceRef = this.getResourceId(path);
+      const corsMethodLogicalId = this.provider.naming
+        .getLogicalApiGatewayMethodName(resourceName, 'options');
```

Comment:
Should we make the names more expressive and let them follow a naming schema?

IMHO here I can't see exactly get at the first glance. What does `logical` exactly mean?

Maybe smth. like `getResourceIdForApiGatewayMethod` or `getLogicalIdForApiGatewayMethod`